File: Section1 - Basics/Patterns/pattern9.js
function printDiamond(n) {
	let str = "";
	for (let i = 0; i < n; i++) {
		for (let j = n; j > i + 1; j--) {
			str += " ";
		}
		for (let k = 0; k <= i; k++) {
			str += "* ";
		}
		str += "\n";
	}

	for (let x = n; x > 0; x--) {
		for (let y = n; y > x; y--) {
			str += " ";
		}
		for (let z = 0; z < x; z++) {
			str += "* ";
		}
		str += "\n";
	}
	console.log(str);
}

printDiamond(5);

File: Section1 - Basics/PatternsJava/pattern9.java
void printDiamond(int n) {
    // Your code here
    for(int i=0; i<n; i++) {
        for(int j=n; j>i+1; j--) {
            System.out.print(" ");
        }
        for(int j=0; j<=i; j++) {
                System.out.print("* ");  
        }
        System.out.print("\n");
    }
    for(int i=n; i>0; i--) {
        for(int j=n; j>i; j--) {
            System.out.print(" ");
        }
        for(int j=0; j<i; j++) {
            System.out.print("* ");
        }
        System.out.print("\n");
    } 
}